install_sct.bat

```diff
@@ -8,6 +8,17 @@ rem This option is needed for expanding !git_ref!, which is set (*and expanded*!
 rem See also https://stackoverflow.com/q/9102422 for a further description of this behavior.
 setLocal EnableDelayedExpansion
 
+rem Try to ensure that Git is available on the PATH prior to invoking `git clone` to avoid 'command not found' errors
+rem   - See also: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3912
+rem NB: This *should* be handled by the git installer, and we even have reports of people running git --version
+rem successfully and still getting an error. However, there are perhaps situations where someone has installed git but
+rem hasn't refreshed their terminal. Manually modifying the PATH is a bit of a hacky workaround, especially if Git has
+rem been installed somewhere else, but if this mitigates a user post on the forum, this will save us some dev time.
+PATH=%PATH%;C:\Program Files\Git
+git -version >nul 2>&1 || (

```

Comment:
Is this missing a dash, or is this a peculiarity of Git on Windows?
```suggestion
git --version >nul 2>&1 || (
```

Reply:
Definitely missing a dash! The [example](https://superuser.com/a/595619) I borrowed this from was testing against `javac -version`, and my brain went on autopilot a bit there. :sweat_smile: